pytorch_forecasting/metrics/point.py

```diff
@@ -5,7 +5,8 @@
 import torch.nn.functional as F
 from torch.nn.utils import rnn
 
-from pytorch_forecasting.metrics.base_metrics import MultiHorizonMetric
+from pytorch_forecasting.metrics.base._base_object import _BasePtMetric
```

Comment:
this seems to be an unusued import.

Please ensure you check for unused imports.